join.html

```diff
@@ -70,26 +89,122 @@ <h2 class="title">Curriculum</h2>
+        </div>
     </div>
 
+    <h2 class="title">Events</h2>
+    <div class="row">
+        <div class="col">
+            <img class="img-responsive" src="{{site.baseurl}}/img/undraw/join-events.svg" alt="Man pointing to events on a calendar"/>
+        </div>
+        <div class="col no-gutter">
+            <p>
+                Our special events team hosts lots of <b>stand-alone</b> events and <b>short courses</b>, usually involving <b>collabs with other orgs</b>. We aim to <b>introduce CS</b> to kids and show them what it’s all about! <a class="tla-link" href="{{site.baseurl}}/citylab-cyber">Cyber Day with CityLab</a> is an example of a past event that we’ve had. Interested? Here’s what we do: 
+            </p>
+            <ul>
+                <li>Collab with other clubs and work together to bring CS to kids (logistics+teaching)</li>
+                <li>Make curriculum and bring ideas to the table!</li>
+                <li><b>Flexible schedule and commitment</b>--you can choose what events you’d like to help with and how you help</li>
+            </ul>
+            <a class="button flex-center" href="{{site.general_interest}}"><span class="fa fa-file-alt"></span> General interest Form</a>
+        </div>
+    </div>
+
+    <h2 class="title">Speaker Series</h2>
+    <div class="row">
+        <div class="col no-gutter">
+            <p>
+                For our Speaker Series, we find one-off topics that we talk about with a class to get them interested in computer science and technology. If you want to meet new students and have fun discussions with them about tech, fill out the interest form to get involved! Our current Speaker Series include:
+            </p>
+            <ul>
+                <li><a class="tla-link" href="{{site.baseurl}}/ai-ethics">AI Ethics</a>: discuss ethical issues that might come about from machine learning and artificial intelligence</li>
+                <li>CS Pathways: talk about different career paths that you can pursue with CS knowledge</li>
+            </ul>
+            <a class="button flex-center" href="{{site.general_interest}}"><span class="fa fa-file-alt"></span> General interest Form</a>
+        </div>
+        <div class="col">
+            <img class="img-responsive" src="{{site.baseurl}}/img/undraw/join-speaker.svg" alt="Woman giving a presentation"/>
+        </div>
+    </div>
+
+    <h2 class="text-25x">What would you do?</h2>
+    <p class="col">Check out some of the great work our members are doing!</p>
+    <div class="dev-projects-grid">
+        <div class="dev-project-card">
+            <div class="dev-project-card-content">
+                <h3 class="title dev-project-title text-center">
+                    Emerson (Python I) Booleans and Conditionals PPT
+                </h3>
+            </div>
+            <a class="button dev-button-container" href="{{site.baseurl}}/files/Booleans_and_Conditionals_Emerson_Week_4.pdf">View</a>
```

Comment:
A small comment - with anchor tags, we want the inner text to be descriptive of what the link leads to, which is very important for accessibility (ex people who use screenreaders). Instead of "View", I suggest we do something like "View Presentation" or "View Worksheet"!